src/Microsoft.Health.Fhir.Core.UnitTests/SubstituteHelpers.cs

```diff
@@ -0,0 +1,21 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Microsoft.Health.Extensions.DependencyInjection;
+using NSubstitute;
+
+namespace Microsoft.Health.Fhir.Core.UnitTests
+{
+    public class SubstituteHelpers
+    {
+        public static Func<IScoped<T>> ScopedFunc<T>(T obj)
```

Comment:
Looks pretty similar to:

https://github.com/microsoft/fhir-server/blob/f0776e2471f3340f163e494220b1cfbdf94147ce/src/Microsoft.Health.Fhir.Core.UnitTests/Extensions/CreateMockedScopeExtensions.cs#L13-L18

I think in other places the usage was something like `Method(param1, () => obj.CreateMockScope())`

Reply:
Darn it, I thought we already had an extension for this but I couldn't find it.